Formula/sile.rb

```diff
@@ -198,6 +198,15 @@ def install
     (bin/"sile").write_env_script libexec/"bin/sile", env
   end
 
+  def caveats
+    <<~EOS
+      Sile uses the font Gentium Plus by default. If this font is not installed or an alternative is not specified, you may encounter errors.
+
+      To install Gentium Plus from a Homebrew Cask run:
+        brew install --cask font-gentium-plus
```

Comment:
This is not in the default caskroom. You'll need to tap `homebrew/cask-fonts` first. Alternatively, is there a way for Homebrew to specify an alternative that does exist on macOS?

Reply:
That's something that has been discussed before. I wasn't aware those fonts had to be tapped. I'll close this in favor of finding an alternative font on Mac.

Reply:
There are no metricly compatible alternatives. You can still render documents with any font that does exist on the platform, but you'll get a warning for documents that either don't specify the font at all or ask for Gentium Plus.

For many apps any substitute font that puts letters on the screen would be acceptable, but for a typesetting engine having a specific font with specific metrics matters a lot. You can't just swap in whatever and call it good. This is why most word processors like M$ Word ship with their own fonts to cover the case of default documents needing to render without reflowing differently across platforms.

Homebrew could consider bundling Gentium Plus in this package or supplying it via some other package that would be available in default repos. Baring one of those options, I suggest this caveat is still a good thing to point out how users need to get started. Adding the tap into the caveat should be better than nothing.

Reply:
https://github.com/silnrsi/font-gentium/blob/master/OFL-FAQ.txt suggests it can just be packaged into Sile though. That will fix the problem for everyone building from source.

Reply:
We've looked into that but decided against it to date. Bundling fonts in our install would cause conflicts with other packages on systems that *do* provide packages and also make the order of precedence even more complicated than it already is (with project, user, and system level fonts) by interjecting app provided ones somewhere in the chain. The mechanism for interjecting (custom `fontconfig` settings) also then blows up end potential other uses users may have for changing their fontconfig settings. We could revisit that and potentially make it an optional install feature, but it isn't a clear win over making platforms handle their own dependencies.